lib/elixir/lib/module.ex

```diff
@@ -801,42 +823,51 @@ defmodule Module do
   developer to customize it. See `Kernel.defoverridable/1` for
   more information and documentation.
   """
-  def make_overridable(module, tuples) do
+  @spec make_overridable(module, [function_arity]) :: :ok
+  def make_overridable(module, tuples) when is_atom(module) and is_list(tuples) do
     assert_not_compiled!(:make_overridable, module)
 
-    :lists.foreach(fn {name, arity} = tuple ->
-      case :elixir_def.take_definition(module, tuple) do
-        false ->
-          raise ArgumentError,
-            "cannot make function #{name}/#{arity} overridable because it was not defined"
-        clause ->
-          neighbours =
-            if :elixir_compiler.get_opt(:internal) do
-              []
-            else
-              Module.LocalsTracker.yank(module, tuple)
+    :lists.foreach(fn
+      {function_name, arity} = tuple
+          when is_atom(function_name) and is_integer(arity) and arity >= 0 and arity <= 255 ->
```

Comment:
do you mean move it to the line above, or indent it to the same level as the line above?